packages/cardano-graphql-services/test/TxSubmit/TxSubmitHttpServer.test.ts

```diff
@@ -0,0 +1,76 @@
+import { TxSubmitHttpServer } from '../../src';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import { getPort } from 'get-port-please';
+import cbor from 'cbor';
+import got from 'got';
+
+const tx = cbor.encode('#####');
```

Comment:
Does this return hex? Seems not much better than `const tx = "abc123"`

Reply:
No, a `Buffer`. I don't strictly need it to be cbor since we're not parsing it on the server, but thought it was better to explicitly demonstrate it.